src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -323,6 +323,46 @@ internal List<WildcardPattern> MatchAll
                 }
             }
 
+            if (!found)
```

Comment:
The cyclomatic complexity of this function is pretty high. Can this be refactored into a smaller function?

Reply:
I could simplify it a bit if we can make the assumption that the psd1 or psm1 file always has the same casing as the folder name.  It's not clear to me if that is a safe assumption or not.  The current code is conservative in that you could have: "...\Modules\FOO\foo.psd1"

Reply:
I'll check with the PSGet folks

Reply:
per PSGet, it is safe to assume the folder and file names have the same casing.  So I'll simplify this code.